core/src/views/dropdown.rs

```diff
@@ -3,11 +3,67 @@ use crate::{
     View,
 };
 
+/// The dropdown icon of a dropdown view.
 pub const ICON_DOWN_OPEN: &str = "\u{e75c}";
 
-pub struct Dropdown {}
+/// A dropdown used to display some state with the ability to open a popup with options to change that state.
+///
+/// Usually a dropdown is used in the context of a "combo box" or "list picker" to allow the user to select
```

Comment:
I could be wrong but I always thought a combo box was specifically a dropdown + textbox combination.

Reply:
I'm pretty sure you could use the Dropdown widget to implement that kind of form element as well!